frontend/src/libs/components/toggle-checkbox/toggle-checkbox.tsx

```diff
@@ -33,11 +33,11 @@ const ToggleCheckbox = <T extends FieldValues>({
         {...field}
         name={name}
         type="checkbox"
-        id={TOGGLE_CHECKBOX_ID}
+        id={name ?? TOGGLE_CHECKBOX_ID}
```

Comment:
I guess name is not optional so we can also get rid of nullish coalescing

Reply:
And that constand in as well I guess